click-to-reveal/script.js

```diff
@@ -0,0 +1,58 @@
+const questions = [
+    {
+        'question': 'Have you heard about the chocolate record player? ',
+        'answer': 'It sounds pretty sweet.'
+    },
+    {
+        'question': 'What do you call a factory that makes okay products?',
+        'answer': 'A satisfactory.'
+    },
+    {
+        'question': 'What did one wall say to the other?',
+        'answer': "I'll meet you at the corner."
+    }
+]
+
+window.addEventListener("DOMContentLoaded", (event) => {
+    console.log("DOM fully loaded and parsed");
+
+   let btnContainer =  document.getElementById("idt-btn-container")
+   for(let i = 0; i < questions.length; i++){
+        btnContainer.innerHTML += `<button>${questions[i].question}</button>`
+   }
+
+    const buttons = document.getElementsByTagName('button');
```

Comment:
```suggestion
    const buttons = btnContainer.getElementsByTagName('button');
```

Grab the buttons from inside the button container, so that we don't accidentally get `<button>` from the general HTML on the page.